grammar.js

```diff
@@ -1485,7 +1452,7 @@ module.exports = grammar({
 
     character: $ => seq(
       "'",
-      repeat(choice(/[^\\']/, $.escape_sequence)),
+      repeat(choice(/[^\\']/, $.escape_sequence, '@')),
```

Comment:
This is a workaround to parse don't look at '@' an expect an decorator inline.

Reply:
The https://github.com/nkrkv/tree-sitter-rescript/pull/218/commits/d6d8aacc87e61167d94cb81633c1570157b108cb has an alternative. I think this approach is better.